chaos_genius/views/anomaly_data_view.py

```diff
@@ -47,6 +45,18 @@ def kpi_anomaly_detection(kpi_id):
     end_date = None
     try:
         kpi_info = get_kpi_data_from_id(kpi_id)
+
+        if not kpi_info["anomaly_params"]:
+            current_app.logger.info(f"No Anomaly Data Found for KPI ID: {kpi_id}")
+            return jsonify(
+                {
+                    "data": "",
+                    "msg": "",
+                    "anomaly_end_date": "",
+                    "last_run_time_anomaly": "",
```

Comment:
Would be better to use `None` (which will become a `null` in JSON) here instead of empty strings:
```suggestion
                    "data": None,
                    "msg": "",
                    "anomaly_end_date": None,
                    "last_run_time_anomaly": None,
```